### PR TITLE
Combine examples and playground pages

### LINKED ISSUE #69: Enhanced Playground
Integrating a drop down that allows to select either from examples or some other sample code.

Continuation of #4

### PR DESCRIPTION
**Pull request by KaneFreeman**
## Description

Combine examples and playground pages into an enhanced playground area with codesandbox.

- Examples page is gone.
- A menu has been added to the playground page with the Sandbox and all of the examples.
- Examples with a codesandbox will show that on their page.
- Examples without a codesandbox will show their demo on their page.
- The menu will be a dropdown menu on desktop and a sliding/scrolling menu on tablet.
- Mobile will not be able to see the Playground menu item on the top menu.

Resolves #69 

### Code

This PR touches:

- [x] Content
- [ ] Content Pipeline
- [x] Frontend
- [ ] Infrastructure

### Tests

- [x] Tests are included?

#### Desktop
image
image
image
image

#### Tablet
image
image
image

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-6rg2ny6y6.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-n6tyz7ucd.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-3r5f2a65w.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-9n9xxfiic.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-p3aemvhi5.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-6igl5absl.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-i9et50adm.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-op60blokk.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-dyrh6460l.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-1ky8muyok.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-c6ryfux17.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-6wq0vw46d.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-qcb4wl95d.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-n1cp1zfma.now.sh

File: package.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
     "build:dev": "dojo build -m=dev -s -w",
     "deploy:now": "npm run build && cp now.json ./output/dist && npx now ./output/dist --public",
     "lint:tslint": "tslint -p .",
-    "lint:prettier": "prettier -l \"{src,tests,scripts}/**/*.{ts,tsx,css}\"",
```
Comment (agubler) [inline package.json]: Do we want to remove the prettier check?

Comment (KaneFreeman) [reply package.json]: Trying to remember why I took this out. Will readd.

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.io: https://dojoio-486538yw3.now.sh
* zh-cn.dojo.io: https://zh-cndojoio-p625ssoh1.now.sh

**Pull request merged**